ocw/lib/EC2.py

```diff
@@ -82,32 +149,21 @@ def cleanup_all(self):
             # 'ImageId': 'ami-00d30c03d17d3db69',
             # 'Name': 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-BYOS-Build1.55.raw.xz'
```

Comment:
this comment should be removed / updated and better also moved into function

Reply:
What I wil show here is, that we get a image  as a dict and the key `Name` contains the name, `ImageId` the id and `CreationDate` the date in that format...

Reply:
What do you think with a command like
# @see https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_images